api/src/context.rs

```diff
@@ -335,17 +335,23 @@ impl Context {
                 E::internal_with_code(err, AptosErrorCode::InternalError, latest_ledger_info)
             })?;
         let block_timestamp = new_block_event.proposed_time();
+
+        // We can only get the max_transactions page size
+        let max_txns = std::cmp::min(
```

Comment:
should we maybe use a partial response in this case (206), to indicate truncation has happened?

Reply:
It's a strange status code in general.  We can look at that in the future